platform/src/hyper/mod.rs

```diff
@@ -108,7 +111,13 @@ pub async fn request<T: Serialize, U: DeserializeOwned>(
     let resp = match client.request(req).await {
         Ok(r) => r,
         Err(err) => {
-            return Err(Error::Remote(err.to_string()));
+
+            // TODO Replace with the correct error
+            let local_error_sc = match StatusCode::try_from(418) {
```

Comment:
I'm not sure what you were planning here yet, but I think the status code is already being returned in Quinn's code below.

Reply:
This came from @qtpeters github branch, he may already have addressed this there